client/src/lib.rs

```diff
@@ -22,38 +22,44 @@
 pub mod att_station;
 pub mod attestation;
 pub mod error;
-pub mod manager;
 pub mod utils;
 
-use crate::manager::NUM_NEIGHBOURS;
-use att_station::{AttestationData as AttData, AttestationStation as AttStation};
-use attestation::{Attestation, AttestationData};
+use crate::att_station::AttestationCreatedFilter;
+use att_station::{AttestationData as ContractAttestationData, AttestationStation as AttStation};
+use attestation::{Attestation, AttestationPayload, SignedAttestation};
 use eigen_trust_circuit::{
-	calculate_message_hash,
-	eddsa::native::{sign, SecretKey},
-	halo2::halo2curves::{bn256::Fr as Scalar, ff::PrimeField},
-	utils::to_short,
-	ProofRaw,
+	dynamic_sets::native::{EigenTrustSet, Opinion},
+	eddsa::native::{sign, PublicKey, Signature},
+	halo2::halo2curves::bn256::Fr as Scalar,
 };
+use error::EigenError;
 use ethers::{
-	abi::Address,
+	abi::{Address, RawLog},
+	contract::EthEvent,
 	prelude::EthDisplay,
-	types::{Bytes, U256},
+	providers::Middleware,
+	signers::Signer,
+	types::{Filter, H256},
 };
 use serde::{Deserialize, Serialize};
-use utils::{setup_client, EtVerifierWrapper, SignerMiddlewareArc};
+use std::collections::HashMap;
+use utils::{
+	ecdsa_eddsa_map, eddsa_sk_from_mnemonic, eth_wallets_from_mnemonic, setup_client,
+	SignerMiddlewareArc,
+};
 
-#[derive(Debug)]
-pub enum ClientError {
-	DecodeError,
-	ParseError,
-	TxError,
-}
+/// Max amount of participants
+const CIRCUIT_SIZE: usize = 2;
```

Comment:
The correct name for this would be `MAX_NEIGHBOURS`.
This constant does affect the circuit size, but so dies the `NUM_ITERATIONS`

Reply:
Thanks! Fixed the name and pushed now.